openedx/core/djangoapps/credentials/tests/test_signals.py

```diff
@@ -23,142 +16,6 @@
 SIGNALS_MODULE = 'openedx.core.djangoapps.credentials.signals'
 
 
-@ddt.ddt
```

Comment:
These methods moved to the `tasks.py` file, so their tests have been moved to the `test_tasks.py` file.